### DISCUSSION TITLE
WebdriverIO error out after clicking on a button

### DISCUSSION THREAD

**Discussion by ryan-cubic** [webdriverio/webdriverio#11641] [Q&A]
WebdriverIO errors out after clicking on a button and fails the test.  There is no obvious reason for it to fail because I can watch it run the test from the Chrome browser.  It clicks on the button (a save button, for example) and the web page saves the form data and navigates to the next expected page.  So the click action flow works.  However why WebdriverIO errors out.

Sample errors:
-Error: Can't call elementClick on element with selector ".ui-button-text=Create Document" because element wasn't found

-Error: invalid selector: The selector "" used with strategy "css selector" is invalid! For more information on selectors visit the WebdriverIO docs at: https://webdriver.io/docs/selectors

NOTE: our web app is developed with https://www.primefaces.org/ library and java ().  Interestingly hundreds of other buttons in the app work fine with WebdriverIO and only certain buttons would not work.  This issue appears to be reproducible more often with Chrome browser.  I tried browser.pause() or refresh() page before clicking on the button but not able to resolve the issue.

Any insights would be appreciated.  Thanks.

**Comment by christian-bromann** [comment]
Duplicate of #11642
